.github/workflows/tests.yml

```diff
@@ -0,0 +1,155 @@
+name: tests
+on:
+  push:
+    branches-ignore: [2.0]
+  workflow_dispatch:
+
+jobs:
+  # test-browser:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: browser-actions/setup-chrome@v1
+  #       with:
+  #         chrome-version: stable
+  #       id: setup-chrome
+  #     - run: |
+  #         echo Installed Chrome version: ${{ steps.setup-chrome.outputs.chrome-version }}
+  #         ${{ steps.setup-chrome.outputs.chrome-path }} --version
+  #
+  #         CHROME_VERSION=$(echo "${{ steps.setup-chrome.outputs.chrome-version }}" cut -f 1 -d ' ' | cut -d '.' -f 1)
+  #
+  #         CHROMEDRIVER_VERSION=$(curl --location --fail --retry 10 http://chromedriver.storage.googleapis.com/LATEST_RELEASE_$CHROME_VERSION)
```

Comment:
this was working a few weeks ago when 114 was in the stable channel:
https://github.com/jsforce/jsforce/actions/runs/5488741617/job/14865068976

current stable is 115 and the URL to get the chromedriver changed:
https://chromedriver.chromium.org/downloads

Will log an issue in the setup-chromedriver action repo.

Reply:
also tried pinning to 114 in setup-chrome (then setup-chromedriver will try to get the driver for 114) but was unable to find the commit reference to chrome 144 (action doesn't accept a version).

Reply:
I saw this issue about that https://github.com/browser-actions/setup-chrome/issues/216

Reply:
Maybe this is an unusual time, but if not, would it make sense to put those urls/commit refs as GHA vars so it's easy to change them later?